components/automate-ui/src/app/page-components/projects-filter/projects-filter.component.html

```diff
@@ -5,5 +5,6 @@
   [selectionCountVisible]="projectsFilter.selectionCountVisible$ | async"
   [selectionCountActive]="projectsFilter.selectionCountActive$ | async"
   [dropdownCaretVisible]="projectsFilter.dropdownCaretVisible$ | async"
+  [filterVisible]="projectsFilter.filterVisible$ | async"
```

Comment:
I suspect you will need to set this in some unit tests for projects-filter-dropdown.component.ts, too.

Reply:
I did, thank you!